Packs/ServiceNow/ReleaseNotes/2_5_24.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ServiceNow v2
+- Fixed an issue where the ***servicenow-get-ticket-notes*** command would fail on tickets with notes that contain two or more new lines.
```

Comment:
```suggestion
Fixed an issue where the ***servicenow-get-ticket-notes*** command would fail on tickets with notes that contain two or more new lines.
```